content/help/addons/groups/managing-groups.md

```diff
@@ -1,28 +0,0 @@
----
-title: Managing Groups
-tags:
-- Features
-- Addons
-- Groups
-category: help
-menu:
-  help:
-    parent: addons-groups
-aliases:
-- /addons/groups/managing-groups
```

Comment:
This alias is missing from the new page.

Reply:
Fixed